python/ray/utils.py

```diff
@@ -70,7 +70,7 @@ def push_error_to_driver(worker,
             will be serialized with json and stored in Redis.
     """
     if driver_id is None:
-        driver_id = ray_constants.NIL_JOB_ID.id()
+        driver_id = ray.ObjectID.nil_id().id()
```

Comment:
It looks like we can change `driver_id` to be a `ray.ObjectID` instead of handling raw bytes.

Reply:
@stephanie-wang I have changed `driver_id` to `ray.ObjectID`. For the question about pickling `ObjectID`, @raulchen had discussed with @robertnishihara , we decided to give it a try. From current Jenkins and Travis test, it works fine. We need to monitor it continuously to see whether users will have problems or could there be difficult bugs when it's pickleable.